Userland/Utilities/gron.cpp

```diff
@@ -41,16 +41,11 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     args_parser.add_positional_argument(path, "Input", "input", Core::ArgsParser::Required::No);
     args_parser.parse(arguments);
 
-    RefPtr<Core::File> file;
-
-    if (path.is_null())
-        file = Core::File::standard_input();
-    else
-        file = TRY(Core::File::open(path, Core::OpenMode::ReadOnly));
+    auto file = TRY(Core::Stream::File::open(path, Core::Stream::OpenMode::Read));
```

Comment:
This no longer read from STDIN if no path is null I think

Although using `open_standard...` technically also changes behavior as `-` would be treated as STDIN